packages/components/organisms/f-header/test/component/f-header.component.mobile.spec.js

```diff
@@ -40,6 +42,7 @@ describe('Mobile - f-header component tests - @browserstack', () => {
             // Act
             header.open(headerData);
             header.waitForComponent();
+            console.log(link);
```

Comment:
Any particular reason for this log?

Reply:
my thinking here was that we need to have some sort of error handling for the navigation links - we're not using the `it` statement in this forEach so I thought a console.log might be the easiest way to do this for now? Let me know what you think though